detection_rules/packaging.py

```diff
@@ -318,7 +318,7 @@ def get_summary_rule_info(r: Rule):
             return rule_str
 
         def get_markdown_rule_info(r: Rule, sd):
-            rules_dir_link = f'https://github.com/elastic/detection-rules/tree/v{self.name}/rules/{sd}/'
+            rules_dir_link = f'https://github.com/elastic/detection-rules/tree/{self.name}/rules/{sd}/'
```

Comment:
we don't include `v` in branch names

Reply:
tree works for tags. so i think this is right?
tags have `major.minor.patch` vs release branches which are `major.minor`

Reply:
oh so maybe that is why I did it - to tie it to the tag and not the release branch. I will revert this then - thanks!

Reply:
sweet. also `sd` is probably not the most descriptive name for a var.
we can come back to this and add an inline comment for the change

Reply:
🙂